README.md

```diff
@@ -57,7 +57,7 @@ Starting local Kubernetes cluster...
 
 ```
 ### Prepare Demo application
-We're going to use a simple NodeJS application. The image is pushed to a DockerHub already (evalle/gordon:v1.0) but you should never-ever download and run unknown images from DockerHub, so let's build it instead.
+We're going to use a simple NodeJS application. The image is pushed to a DockerHub already (${REGISTRY_USER}/gordon:v1.0) but you should never-ever download and run unknown images from DockerHub, so let's build it instead.
```

Comment:
do some export of this variable so it can be used easily and make it default to evalle? or push the image as prgcont?

Reply:
The thing is that we already have the image (evalle/gordon:v1.0) and it publically available so everyone can use it, but it's better to build it and I describe how to do it in the following description. I don't see why we need this change.

Reply:
I would slightly prefer prgcont/gordon:v1.0 :) If you ping me you dockerhub ID I'll add you to do org.. but I agree is so minor. 

The purpose of this change is that we tell people to not run random stuff from dockerhub. So we should  to eat our own dogfood :)